pycti/entities/opencti_malware_analysis.py

```diff
@@ -436,7 +436,7 @@ def create(self, **kwargs):
                         "hostVm": hostVm,
                         "operatingSystem": operatingSystem,
                         "installedSoftware": installedSoftware,
-                        "sample": sample,
+                        "analysisSample": sample,
```

Comment:
this is a lazy update... the client API is unchanged just the mapping to call the backend API.
Happy to discuss if we need to go beyond that and break also the client api to be more consistent